parl/remote/scripts.py

```diff
@@ -85,9 +85,7 @@ def cli():
 @click.command("start", short_help="Start a master node.")
 @click.option("--port", help="The port to bind to.", type=str, required=True)
 @click.option(
-    "--debug",
-    help="Start parl in debug mode to show all logs.",
-    default=False)
+    "--debug", help="Start parl in debug mode to show all logs.", is_flag=True)
```

Comment:
Start parl in the debugging mode to print all running log.

Reply:
done